src/thematic/1_water/sfp-105-nl_bemonsteren-peilbuis/04_principe.Rmd

```diff
@@ -0,0 +1,33 @@
+# Principe
+
+```{=html}
+<!--
+Korte omschrijving van de principes waarop het protocol gebaseerd is.
+Bv principe achter het nemen van mengmonsters, principe achter visueel inschatten van de bedekking van een plant, ...
+-->
+```
+Het doel is een staal te nemen van 'vers' grondwater.
+Dat betekent dat er geen monsters mogen genomen worden van grondwater dat weken of maanden in het meetpunt stil staat.
+De groei van micro-organismen zorgt er immers voor dat grote hoeveelheden metabolieten in het water accumuleren.
+Daarom wordt de wand van de piezometer eerst gereinigd en daarna wordt het water uit de piezometer weggepompt.
+
+Het bemonsteren van de piëzometer gebeurt 1 tot 3 dagen na het reinigen.
+Het 'vers' grondwater wordt opgepompt.
+De zuurtegraad (pH) en geleidbaarheid (EC) worden op het terrein gemeten en de recipiënten voor chemische analyse in het laboratorium worden gevuld.
+Het aantal deelmonsters is afhankelijk van de chemische variabelen die zullen worden gemeten.
+De richtlijnen voor het nemen en van deelmonsters en het conserveren van de deelmonsters zijn terug te vinden in SOP-005.
```

Comment:
SOP-005: in afwachting van publicatie via protocolsource, kan je hier verwijzen naar https://sites.google.com/inbo.be/veldprotocollen/standaard-veld-procedures-svp/0-generiek